crates/ironrdp-svc/src/lib.rs

```diff
@@ -22,35 +22,35 @@ use std::marker::PhantomData;
 /// The integer type representing a static virtual channel ID.
 pub type StaticChannelId = u16;
 
-/// SVC data which sould be sent to the server over the main communication channel (RDP socket).
+/// SVC data to be sent to the server. See [`SvcMessage`] for more information.
 /// Usually returned by the channel-specific methods.
-pub struct SvcRequest<C> {
+pub struct SvcMessagesWithProcessor<P: StaticVirtualChannelProcessor> {
```

Comment:
I feel this type name is a bit too verbose, but I understand where you come from. I would suggest to change `With` into `For`, because the processor is not held
```suggestion
pub struct SvcMessagesForProcessor<P: StaticVirtualChannelProcessor> {
```